### DISCUSSION TITLE
Restrict File Types

### DISCUSSION THREAD

**Discussion by Kuuchuu** [filebrowser/filebrowser#1872] [Ideas]
Unless I've somehow missed it and it's already a thing, being able to restrict a user to only see/access/upload specific file types would be useful.

**Comment by Kuuchuu** [comment]
Being able to restrict directory creation separate from file creation would also be very useful.

**Comment by TomLewis** [comment]
Came here to request the same basic feature.

**Comment by ramiresviana** [comment]
Rules can be used to control access of the files, check this example: https://github.com/filebrowser/filebrowser/issues/1948
